paddle/fluid/operators/mkldnn/softmax_mkldnn_op.cc

```diff
@@ -74,22 +74,34 @@ class SoftmaxMKLDNNHandler
     }
   }
 
-  SoftmaxMKLDNNHandler(const std::vector<int64_t>& dims,
-                       const MKLDNNMemoryFormat fmt,
-                       const MKLDNNMemoryFormat diff_fmt, const int& axis,
+  SoftmaxMKLDNNHandler(const paddle::framework::ExecutionContext& ctx,
                        const platform::MKLDNNDeviceContext& dev_ctx,
```

Comment:
```suggestion
                       const MKLDNNDeviceContext& dev_ctx,
```
Since you have using "paddle::platform::MKLDNNDeviceContext;" in line 33, you don't need to declare this namespace here

Reply:
good catch